1-mandatory/2-people-i-know.js

```diff
@@ -393,16 +396,30 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+let friendsWhoAreColleaguesOfStacie = friends
+.filter(friend => friend.colleagues
+.find(friend => friend.name === "Stacie Villarreal"))
+.map(friend => `${friend.name.first} ${friend.name.last}`);
```

Comment:
This is amazing. 

Pro tip is that if we use VSCode to format the code it'll look even nicer https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode